deploy/mysql-pv.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: mysql-pv-volume
+  labels:
+    type: local
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 20Gi
```

Comment:
Maybe we need to reduce the storage size since we will not allocate so much disk space to Minikube node.

Reply:
has been modified to 1G